dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -150,24 +154,24 @@ public Response organizationsGet(SecurityContext securityContext) {
     @Override
     public Response toolsIndexGet(SecurityContext securityContext) {
         List<Entry> published = getPublished();
-        if (!config.getEsConfiguration().getHostname().isEmpty() && !published.isEmpty()) {
+        if (!config.getEsConfiguration().getHostname().isEmpty()) {
 
             try (RestClient restClient = RestClient
                     .builder(new HttpHost(config.getEsConfiguration().getHostname(), config.getEsConfiguration().getPort(), "http"))
                     .build()) {
-
+                LOG.info("Deleting the ES index.");
                 // Delete index
                 try {
                     restClient.performRequest("DELETE", "/entry");
                 } catch (Exception e) {
-                    e.printStackTrace();
+                    throw new CustomWebApplicationException(e.getMessage(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
We often start with a new elasticsearch container (with no index).  This makes the endpoint unusable.

Reply:
Should probably have a LOG.error to replace the printStackTrace

Reply:
I think the issue Gary is pointing out, is that we hit https://staging.dockstore.org/static/swagger-ui/index.html#/extendedGA4GH/toolsIndexGet when deploying a new version with an empty ES.
If this throws an exception then we won't be able to do that.